src/wfrpCampaignManager/domain/dataModel/armourDataModel/ArmourDataModel.js

```diff
@@ -0,0 +1,15 @@
+class ArmourDataModel {
+  constructor({ name, price, encumbrance, availability, penalty, locations, armourPoints, qualitiesAndFlaws, group }) {
+    this.name = name;
```

Comment:
Recall that if you use "_" for properties name, they will be private and I think that is the intention here ;)
```suggestion
    this._name = name;
```